spec/controllers/account/users_controller_spec.rb

```diff
@@ -0,0 +1,29 @@
+require 'rails_helper'
+
+RSpec.describe Account::UsersController, type: :controller do
+  let!(:user) { create(:user) }
+
+  before do
+    sign_in user
+  end
+
+  describe "GET #edit" do
+    it "must display edit page" do
+      get :edit, params: { user_id: user.to_param, id: user.to_param }
```

Comment:
for profile editing URL must be /profile/edit

if I see anything else in URL - task is not implemented